src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -498,7 +522,13 @@ public void Initialize()
                 };
 
                 // read all script functions and apply to JobHostConfiguration
-                Collection<FunctionDescriptor> functions = GetFunctionDescriptors(functionMetadata);
+                Collection<FunctionDescriptor> functions;
+                using (metricsLogger.LatencyEvent(MetricEventNames.HostStartupGetFunctionDescriptorsLatency))
```

Comment:
This doesn't seem like the right scope to capture. I think we also want to include the FunctionGenerator.Generate call below which is where we IL gen the wrappers. All of this is then a logical "functions initialized" step

Reply:
I see 500ms being spent on this exact step. I don't want to that to get mixed in with IL gen.